.github/workflows/pull.yml

```diff
@@ -469,16 +469,16 @@ jobs:
         ]}
     secrets: inherit
 
-  linux-focal-cuda12_6-py3_10-gcc9-sm89-test:
-    name: linux-focal-cuda12.6-py3.10-gcc9-sm89
+  linux-focal-cuda12_6-py3_10-gcc11-sm89-test:
+    name: linux-focal-cuda12.6-py3.10-gcc11-sm89
     uses: ./.github/workflows/_linux-test.yml
     needs:
-      - linux-focal-cuda12_6-py3_10-gcc9-sm89-build
+      - linux-focal-cuda12_6-py3_10-gcc11-sm89-build
       - target-determination
     with:
-      build-environment: linux-focal-cuda12.6-py3.10-gcc9-sm89
-      docker-image: ${{ needs.linux-focal-cuda12_6-py3_10-gcc9-sm89-build.outputs.docker-image }}
-      test-matrix: ${{ needs.linux-focal-cuda12_6-py3_10-gcc9-sm89-build.outputs.test-matrix }}
+      build-environment: linux-focal-cuda12.6-py3.10-gcc1-sm89
```

Comment:
Gcc1 - typo

Reply:
Supporting gcc1 was a special request from Dr Emmet Brown who is looking forward to build  PyTorch from source back in 1987